src/firewheel/control/ansible_playbooks/roles/firewheel/tasks/git_clone_loop.yml

```diff
@@ -0,0 +1,45 @@
+---
+###################################################################################################
+# This playbook will clone a Git repository and move specified cached files
+# If you are using an access token, you can specify it in the `server_url`.
+# For example: https://<token>@github.com/user/repo.git
+# The user will need to provide all relevant information.
+# An example configuration is shown below:
+#
+# git_servers:
+#   - server_url: "https://github.com"
+#     repositories:
+#       - path: "firewheel/mc_repo1"
+#       - path: "firewheel/mc_repo2"
+#         branch: "develop"
+#   - server_url: "ssh://git@gitlab.com"
+#     repositories:
+#       - path: "emulytics/firewheel/mc_repo3"
+#         branch: "feature-branch"
+#   - server_url: "https://user:ACCESS-TOKEN@github.com/"
+#     repositories:
+#       - path: "firewheel/mc_repo4"
+###################################################################################################
+
+- name: Clone the repository
+  ansible.builtin.git:
+    repo: "{{ firewheel_server_url }}/{{ firewheel_repo_path }}"
+    dest: "{{ ansible_remote_tmp }}/git"
+    clone: true
+    version: "{{ firewheel_repo_branch }}"
+    update: true
+  register: git_clone_result
```

Comment:
Per the suggestion from @gregjacobus we will try to not pull LFS objects unless they are necessary for a specific required file.